collectors/python.d.plugin/sensors/README.md

```diff
@@ -20,7 +20,10 @@ cd /etc/netdata   # Replace this path with your Netdata config directory, if dif
 sudo ./edit-config python.d/sensors.conf
 ```
 
-### possible issues
+Uncomment the `sensors=force` line and save the file. Restart Netdata with `sudo systemctl restart netdata` to enable
+Raspberry Pi temperature sensor monitoring.
```

Comment:
'sensors=force' is needed for bash sensors plugin. And we have that note for it:

https://github.com/netdata/netdata/blob/d2a3975d0e60aec5f8316debfff11500d9480ef8/collectors/charts.d.plugin/sensors/README.md?plain=1#L32-L47

Reply:
I closed the PR, but then shouldn't we suggest in  https://learn.netdata.cloud/guides/monitor/pi-hole-raspberry-pi#enable-temperature-sensor-monitoring that people use the python plugin instead?

Reply:
@cakrit from what I remember there is no `lm-sensors` package for RPi (or it doesn't work). That is the reason we suggest using the bash version.